src/main/java/service/imp/WriterServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package service.imp;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import service.WriterService;
+
+public class WriterServiceImpl implements WriterService {
+    @Override
+    public void write(String resultingReport, String fileName) {
+        try {
+            Files.write(Path.of(fileName), resultingReport.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file" + fileName);
```

Comment:
It's recommended to include the exception object when rethrowing as a RuntimeException to preserve the stack trace. Change this to `throw new RuntimeException("Can't write to file: " + fileName, e);`